components/people/directory/people_list/people_list.tsx

```diff
@@ -83,25 +83,25 @@ const PeopleList = ({
                 <InfiniteLoader
                     ref={infiniteLoaderRef}
                     isItemLoaded={isItemLoaded}
-                    itemCount={itemCount}
+                    itemCount={1000}
                     loadMoreItems={loadMoreItems}
                 >
                     {({onItemsRendered, ref}) => {
                         const columnWidth = 320;
-                        const rowHeight = 365;
-                        const columnCount = width / columnWidth;
+                        const rowHeight = 320;
+                        const columnCount = Math.floor(width / columnWidth);
+
                         return (
                             <FixedSizeGrid
                                 ref={ref}
-                                itemData={people}
+                                itemData={{width}}
```

Comment:
Is this intended?

Reply:
Yep that's what it's for. Don't need the people data passed through this prop